### ISSUE TITLE
">>>" raises an error when encountered in rst files.

### ISSUE THREAD

**Issue by KelSolaar** [keewis/blackdoc#165]
Hello,

I'm starting to test `blackdoc` and I came across this issue with [one of our RST](https://github.com/colour-science/colour/blob/develop/docs/index.rst) files: 

```reStructuredText
Features
--------

Most of the objects are available from the ``colour`` namespace:

.. code-block:: python

    >>> import colour
```

```
error: cannot format /Users/kelsolaar/Documents/Development/colour-science/colour/docs/index.rst: Cannot parse: 40:0: >>> import colour
```

Cheers,

Thomas

**Comment by keewis** [comment]
thanks for the report, @KelSolaar. I think this is a duplicate of #150, which has been fixed in #151 (but not yet released). Can you confirm?

**Comment by keewis** [comment]
same as in #166: this should be fixed in the new release
